lib/Client.js

```diff
@@ -7,9 +7,22 @@ var min = require('osc-min');
 var Message = require('./Message');
 
 var Client = function (host, port) {
+    var isBroadcast = host.split('.').reduce(function(previous, current){
+        return previous ? true : +current === 255;
+    }, false),
+    setter = function () {
+        this.setBroadcast(isBroadcast);
```

Comment:
this is super confusing... the amount of binding going around is very confusing. 

Why not just 

```
if (isBroadcast) {
    this._sock.setBroadcast(true)
}
```

or somthing similar.